pyvisa_py/tcpip.py

```diff
@@ -692,6 +725,9 @@ def _set_tcpip_keepalive(
             self.interface.setsockopt(
                 socket.SOL_SOCKET, socket.SO_KEEPALIVE, 1 if attribute_state else 0
             )
+            self.interface.sock.setsockopt(socket.IPPROTO_TCP, socket.TCP_KEEPIDLE, 60)
```

Comment:
Since the setting is of the other options is not conditional on the value (I assume this is fine), it may be worth doing it earlier (and the same would apply for the INSTR too).

Reply:
I'm not sure if it is necessary as this only applies if you enable keepalive packets. I'm also pretty positive, that this function would only be called once in the lifetime of a session.

Reply:
I guess you are right in practice people are unlikely to enable and disable this many times.